internal/docs/mdx.tpl

```diff
@@ -0,0 +1,44 @@
+---
+navigation: "cli"
+title: |-
+{{ .Name }}
+description: |-
+{{ .Description }}
+slug: tools/cli/commands/{{ .Slug }}
+---
+{{ if .SubCommands }}
+{{ range $subCommand := .SubCommands }}
+## {{ $subCommand.Name }}
+
+{{ $subCommand.Description }}
+
+### Usage
+
+`{{ $subCommand.Usage }}`
+
+{{ if $subCommand.Examples }}### Examples{{ end }}
+{{ $subCommand.Examples }}
+{{ range $flagKey, $flagSlice := $subCommand.Flags }}
+{{ if $flagSlice }}### Flags {{ end }}
+{{ range $flag := $flagSlice }}
+- {{ if $flag.Shorthand }}`-{{ $flag.Shorthand }}`, {{ end }}`--{{ $flag.Name }}`: {{ $flag.Description }}
+{{ end }}
+{{ end }}
+{{ end }}
+{{ else }}
+## {{ .Name }}
+### Usage
+
+`{{ .Usage }}`
+
+{{ if .Examples }}
+### Examples
+{{ .Examples }}
```

Comment:
In this case, the examples won't even be wrapped between backticks?
For the new docs, I would also split the examples so we can more easily copy them / run them in the webcli.
Adding a new method to the `Command` struct like that:
```go
type Example struct {
	Desc   string
	Code   string
	WebCLI string
}

func (cmd Command) ExamplesList() []Example {
	var examples []Example
	examplesRaw := strings.Split(cmd.Examples, "#")
	for _, example := range examplesRaw {
		example = strings.TrimSpace(example)
		if len(example) == 0 {
			continue
		}
		exampleLines := strings.Split(example, "\n")

		code := strings.ReplaceAll(exampleLines[1], "$", "")
		code = strings.TrimSpace(code)

		examples = append(examples, Example{
			Desc: strings.Replace(exampleLines[0], "#", "", 1),
			Code: code,
		})

		// TODO: Add WebCLI example depending of:
		// 1. The presence of the `runInWebCLI` annotation on the command
		// 2. The presence of any piping or file flags in the example
	}
	return examples
}
```

WDYT?